modules/layers/src/layers/editable-geojson-layer.ts

```diff
@@ -403,6 +398,7 @@ export default class EditableGeoJsonLayer extends EditableLayer<
     let selectedFeatures = [];
     if (Array.isArray(props.selectedFeatureIndexes)) {
       // TODO: needs improved testing, i.e. checking for duplicates, NaNs, out of range numbers, ...
+      // @ts-ignore
```

Comment:
We typically use ts-expect-error as that version warns when this is no longer needed.

Reply:
As i see this is caused by changes in deckGL types, data can be string or even Promise.
Maybe in will be safer to do something like this instead
```ts
if (typeof props.data === 'object' && 'features' in props.data) {
  const propsData = props.data as { features: Record<string, any> };
  selectedFeatures = props.selectedFeatureIndexes.reduce((acc, elem) => {
    if (propsData.features[elem]) {
      acc.push(propsData.features[elem]);
    }
    return acc;
  }, []);
} else {
  // eslint-disable-next-line no-console
  console.debug(`Cant't select features - state have unexpected shape`);
}
```